### ISSUE TITLE
Wound undetected by common-healing

### ISSUE THREAD

**Issue by jandersson** [rpherbig/dr-scripts#1609]
```
[combat-trainer]>health                                            
Your body feels slightly battered.                                 
Your spirit feels full of life.                                    
You are slightly fatigued.                                         
You have a bruised, swollen and bleeding left eye.                 
                                                                   
Bleeding                                                           
            Area       Rate                                        
-----------------------------------------                          
        left eye       (tended)                                    
>                                                                  
[combat-trainer: Severity to Wounds: {}]                           
```

**Comment by jandersson** [comment]
I think `and bleeding` gets appended to the text if its bleeding?

**Comment by jandersson** [comment]
Not sure why this is undected, I copied the regex and game string into rubular and its detected:
http://rubular.com/r/5DHPXsT1Ph

**Comment by jandersson** [comment]
Found the bug. The wound line is split on commas and this wound contains a comma.
line 166 `wounds_line.split(',').each do |wound| `

**Comment by rpherbig** [comment]
Fixed by #1610
